packages/rspack/src/watching.ts

```diff
@@ -56,8 +56,7 @@ class Watching {
 		dirs: Iterable<string>,
 		missing: Iterable<string>
 	) {
-		// @ts-expect-error
-		this.pausedWatcher = null;
+		this.pausedWatcher = undefined;
```

Comment:
```suggestion
		this.pausedWatcher = void 0;
```

Reply:
This trick is not necessary, bundler will help you do that.